.github/workflows/test-vurnabilities-data.yml

```
@@ -0,0 +1,124 @@
+                client.release();
+                return extractedData; // Return the extracted data
+              } catch (err) {
+                console.error('Error fetching vulnerability data:', err);
+                client.release();
+              }
+            })();
+              
+      - name: Upload Vulnerability Data
+        uses: actions/upload-artifact@v3
+        with:
+            name: vulnerability-data
```

yamllint

[warning] wrong indentation: expected 10 but found 12

(indentation)